.devops/deploy-pipelines.yml

```diff
@@ -98,24 +98,13 @@ stages:
             inputs:
               SourceFolder: '$(System.DefaultWorkingDirectory)'
               TargetFolder: '$(System.DefaultWorkingDirectory)/bundle'
+              # The list of files to be considered is determined by the "files" entry in package.json
```

Comment:
what do you mean?

Reply:
this: https://github.com/pagopa/io-functions-commons/blob/ca9a943a9004bbf7aec7804c5715fbff80ab8fde/package.json#L8

Reply:
yes, I saw it after this comment.. sorry about that!